examples/2d/texture_atlas.rs

```diff
@@ -78,8 +78,10 @@ fn setup(
             scale: Vec3::splat(4.0),
             ..default()
         },
-        sprite: TextureAtlasSprite::new(vendor_index),
-        texture_atlas: atlas_handle,
+        atlas: TextureAtlas {
+            index: vendor_index,
+            layout: atlas_handle,
+        },
```

Comment:
```suggestion
        },
        texture: texture_atlas_texture.clone(),
```

omitting the texture means the sprite silently fails to render here.

we should probably set `Handle<Image>::default` to a "missing pink" or something (it seems it does still display if the rect is out of the image bounds).

Reply:
So in the render process or in the `Default` impl of texture atlas bundles ?

Reply:
i think it's worth doing globally, using `Handle<Image>::default` is probably always a mistake.

but it should be a separate pr, there might be some subtleties around the asset load state, or maybe some other part of the code where default gets interpreted differently maybe.